docs/docs/migration.md

```diff
@@ -76,11 +76,10 @@ If you used this builder for other module types, you can repace it with the `npm
 
 ---
 
-<li> 
+<li> Service creation or service binding parameters provided in external files and referenced by the `path` property of the correponding entity in the `mta.yaml`are packaged differently by the tools into the result MTA archive.
```

Comment:
I'm not sure why but the "path" and "mta.yaml" aren't formatted correctly even though they have the ` characters around them